workloads/whereabouts/metrics-profiles/hosted-cluster-metrics.yml

```diff
@@ -0,0 +1,106 @@
+# Hosted cluster metrics
```

Comment:
Do we care to have this? Not testing multus w/ Hosted Clusters, right?

Reply:
away it goes